src/components/Badge.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Badge({type, checked, handleCheckboxChange }) {
+  return (
+    <>
+      <input className='mx-2 focus:outline-none' type={type} checked={checked} onChange={handleCheckboxChange} />
```

Comment:
why input is used for badge

Reply:
in input i have set type checked

Reply:
then this should be for checkbox. right ? not for the badge